__tests__/app.test.js

```diff
@@ -28,10 +28,49 @@ describe("/api/categories", () => {
     });
 });
 
+describe("/api/reviews/:review_id", () => {
+    test("GET 200: should response with a review object", () => {
+        return request(app)
+            .get("/api/reviews/3")
+            .expect(200)
+            .then((response) => {
+                const { review } = response.body;
+                expect(review).toMatchObject({
```

Comment:
if you're getting back the review with the id of 3, we know exactly what this review should look like and don't have to use expect.any(String) etc, we know what the strings are